### PR TITLE
Update readme and other small changes

### PR DESCRIPTION
**Pull request by bechurch**
**Summary**

This PR fixes/implements the following **bugs/features**

* [x] Updates the Readme to use `cp` instead of `mv` to adding tracked changes during setup
* [x] Removes codeclimate from readme as its broken and unused. (Blessed by Brooke)
* [x] Rename `/.github` to `/docs`
* [x] Move all github templates and example files to `/docs`
* [x] Add migration steps to Readme
* [x] Fixes `brew` typo `service` -> `services`
* [x] Adds me as a maintainer 🎉 
* [x] Adds steps to get credentials from local service

### CODE REVIEW

**Comment by expede** [comment]
I'm not sure if GH will like renaming `.github` 🤔 `docs` is generally used fro generated documentation, AFAIK. Also, from the [GH docs](https://help.github.com/en/articles/about-issue-and-pull-request-templates):

> Issue templates are stored on the repository's default branch, in a hidden .github/ISSUE_TEMPLATE directory

Have you had success with moving them to a `docs` directory before? Thoughts?

File: .codeclimate.yml [hunk 1]
```diff
@@ -1,6 +0,0 @@
-engines:
-  hlint:
-    enabled: true
-ratings:
-  paths:
-  - "**.hs"
```
Comment (expede) [inline .codeclimate.yml]: We actually do have CodeClimate running the linter:

https://codeclimate.com/github/fission-suite/ipfs-api/issues

File: fission-web-api.cabal [hunk 2]
```diff
@@ -4,17 +4,19 @@ cabal-version: 2.2
...
 homepage:       https://github.com/fission-suite/web-api#readme
 bug-reports:    https://github.com/fission-suite/web-api/issues
 author:         Brooklyn Zelenka,
-                Daniel Holmgren
+                Daniel Holmgren,
+                Ben Church
 maintainer:     brooklyn@fission.codes,
-                daniel@fission.codes
+                daniel@fission.codes,
+                ben@fission.codes
```
Comment (expede) [inline fission-web-api.cabal]: 🎉

File: README.md [hunk 3]
```diff
@@ -14,25 +13,66 @@ in a familiar, compatible way
...
+## Setup your database
+```shell
+# Create the database
+createdb web_api
+
+# Apply the projects database schema
+stack repl
+λ> import Fission.Storage.PostgreSQL.Migrate
+λ> import Database.Selda.PostgreSQL
+λ> sequence_ $ mutations "localhost" 5432 "web_api" Nothing Nothing Nothing
```
Comment (expede) [inline README.md]: 👍

File: README.md [hunk 4]
```diff
@@ -14,25 +13,66 @@ in a familiar, compatible way
...
+           "region"       : "amazon-web-services::ap-northeast-1"
+         }' | json_pp
+```
+
+## Send your first request
+```shell
 curl \
   -H "Content-Type: text/plain;charset=utf-8" \
-  -H "Authorization: Basic Q0hBTkdFTUU6U1VQRVJTRUNSRVQ=" \
+  -u "FISSION_USERNAME:FISSION_PASSWORD" \
```
Comment (expede) [inline README.md]: 🙏 Thanks!

**Comment by expede** [comment]
NOTE: I hit the `Update` button, which merged the new `master` in here

File: .codeclimate.yml (hunk 1 above)
Comment (bechurch) [reply .codeclimate.yml]: hmm ok, Ill try fixing that label then :)

**Comment by bechurch** [comment]
@expede On the note of `/docs` vs `/.github`:

I think you make a good point of it being a typical path for generated `docs` on that alone I think theres grounds to leave it as `/.github` and find another place for `/example-files`.

`/docs` does works as a replacement though.

image
https://help.github.com/en/articles/about-issue-and-pull-request-templates#pull-request-templates

Let me know _your_ preference! :)

File: .codeclimate.yml (hunk 1 above)
Comment (bechurch) [reply .codeclimate.yml]: nvm I see why thats returning a `?` now.

File: README.md [hunk 5]
```diff
@@ -4,7 +4,6 @@
 
 [![Build Status](https://travis-ci.org/fission-suite/web-api.svg?branch=master)](https://travis-ci.org/fission-suite/web-api)
 [![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://github.com/fission-suite/blob/master/LICENSE)
-[![Maintainability](https://api.codeclimate.com/v1/badges/44fb6a8a0cfd88bc41ef/maintainability)](https://codeclimate.com/github/fission-suite/web-api/maintainability)
```
Comment (expede) [inline README.md]: 👍

File: package.yaml [hunk 6]
```diff
@@ -4,9 +4,11 @@ category: API
 author:
   - Brooklyn Zelenka
   - Daniel Holmgren
+  - Ben Church
 maintainer:
   - brooklyn@fission.codes
   - daniel@fission.codes
+  - ben@fission.codes
```
Comment (expede) [inline package.yaml]: 🎉

**Comment by expede** [comment]
> CODE_OF_CONDUCT.md → .github/CODE_OF_CONDUCT.md 

We should make these changes to https://github.com/fission-suite/project-template as well

**Pull request merged**